src/Tizen.NUI.Components/Controls/Slider.Internal.cs

```diff
@@ -623,6 +623,51 @@ private int BgTrackLength()
             return bgTrackLength;
         }
 
+        private int BgTrackOffset()
+        {
+            int bgTrackOffset = 0;
+            IndicatorType type = CurrentIndicatorType();
+
+            if (type == IndicatorType.None)
+            {
+                if (direction == DirectionType.Horizontal)
+                {
+                    bgTrackOffset = (int)(thumbImage.Size.Width * 0.5f);
+                }
+                else if (direction == DirectionType.Vertical)
+                {
+                    bgTrackOffset = (int)(thumbImage.Size.Height * 0.5f);
+                }
+            }
+            else if (type == IndicatorType.Image)
```

Comment:
Code on Image and Text seems very similar so could be merge them one single bracket by changing Size of type.

Reply:
done.